test/OpenTelemetry.Tests/Implementation/Trace/ActivityExtensionsTest.cs

```diff
@@ -107,5 +107,24 @@ public void LastSetStatusWins()
 
             Assert.True(activity.GetStatus().IsOk);
         }
+
+        [Theory]
+        [InlineData(ActivityKind.Client, ActivityKind.Client)]
```

Comment:
why do need to pass the expected kind? Its same as input right?

Reply:
I will change soon. :)